lib/tasks/work_tasks.rake

```diff
@@ -25,4 +25,14 @@ namespace :work do
     end
   end
 
+  # AO3-6065
```

Comment:
We'll be able to check the history or blame if we ever need to know what issue this task was for (as opposed to what it does, which is covered in the description), so this can be left off.